enterprise/utils.py

```diff
@@ -1471,7 +1471,7 @@ def enroll_licensed_users_in_courses(enterprise_customer, licensed_users_info, d
         user = User.objects.filter(email=licensed_user_info['email']).first()
         try:
             if user:
-                succeeded = enroll_user(enterprise_customer, user_email, course_mode, course_run_key)
+                succeeded = enroll_user(enterprise_customer, user, course_mode, course_run_key)
```

Comment:
aargh, I hate that the docstring for this method does not indicate types!! if you are willing to, pls update that with types too. https://github.com/edx/edx-enterprise/blob/8625b3a3753ea0c2315b1cd3c2a8521b45c306e9/enterprise/utils.py#L1346